libs/fixtures/src/data/electionMultiPartyPrimary/semsFiles/standard.csv.ts

```diff
@@ -1,364 +1,364 @@
-export default `"sample-county","precinct-1","assistant-mayor-contest-liberty","Assistant Mayor","0","Li","0","Write-in","0","Li","30",
-"sample-county","precinct-1","assistant-mayor-contest-liberty","Assistant Mayor","0","Li","1","Times Over Voted","0","Li","30",
-"sample-county","precinct-1","assistant-mayor-contest-liberty","Assistant Mayor","0","Li","2","Times Under Voted","0","Li","30",
+export default `"sample-county","precinct-1","assistant-mayor-contest-liberty","Assistant Mayor","0","Li","0","Write-in","0","NP","30",

```

Comment:
In the past we had a bug where when an election had a party with the id of "0" it got mislabelled as the party in write-in/undervotes/overvotes. This was fixed already in module-converter-sems but the SEMS fixtures files that were autogenerated were never updated after this bug fix.